### PR TITLE
Promote OpenZeppelin contract wizard

### PR DESCRIPTION
**Pull request by kasperski95**
Show link to the OZ contract wizard. It should help newcomers to kick-start the project.

### CODE REVIEW

**Review by Arcticae — CHANGES_REQUESTED** [review]
I don't understand why we need to do this, promoting 3rd party tools is not responsibility of protostar

**Comment by kasperski95** [comment]
@Arcticae Implicit responsibility of Protostar is minimizing resources (human time) needed to create and maintain contracts on StarkNet. This link cost us nothing and helps push new projects in a right direction.

File: protostar/commands/init/init_command.py [hunk 1]
```diff
@@ -73,12 +75,16 @@ def init(
...
+                    "as a project instead of creating a new project?"
                 )
 
         if should_adapt_existing_project:
             self._adapted_project_creator.run()
         else:
+            logger.info(
+                "Consider using OpenZeppelin Contracts Wizard to kickstart your project.\n"
+                "https://wizard.openzeppelin.com/cairo"
+            )
```
Comment (MaksymilianDemitraszek) [inline protostar/commands/init/init_command.py:87]: We definiately shouldn't include such stuff in logs. We can consider adding it to the docs but I am not sure about this either

**Review by MaksymilianDemitraszek — CHANGES_REQUESTED** [review]

**Comment by kasperski95** [comment]
Outvoted

**Pull request closed without merging**